Diary/Model/Diary.swift

```diff
@@ -7,15 +7,11 @@
 
 import Foundation
 
-struct Diary: Codable, Hashable, Identifiable {
-    let id = UUID()
+struct Diary: Hashable, Identifiable {
+    let id: UUID
     let title: String
     let body: String
     let timeIntervalSince1970: Int
     
-    enum CodingKeys: String, CodingKey {
-        case title
-        case body
-        case timeIntervalSince1970 = "created_at"
-    }
+    var sharedText: String { title + "\n\n" + body }
```

Comment:
```suggestion
    var sharedText: String {
        return title + "\n\n" + body 
    }
```

연산 프로퍼티는 한줄로 잘 안쓰는거 같아요!
return 붙이고 안붙이는건 취향차이 같고용ㅎㅎ

Reply:
3d6e6883fe483343075d5a9629a23ead237c10c2
`return` 명시하고, 라인을 분리했습니다!